components/test-suite/src/loci/tests/testng/FormatReaderTest.java

```diff
@@ -1123,6 +1123,18 @@ public void testDeltaT() {
     for (int i=0; i<reader.getSeriesCount(); i++) {
       config.setSeries(i);
 
+      // Test image acquisition date
+      String expectedDate = config.getDate();
+      String date = null;
+      if (retrieve.getImageAcquisitionDate(i) != null) {
+        date = retrieve.getImageAcquisitionDate(i).getValue();
+      }
+      if (expectedDate != null && date != null && !expectedDate.equals(date)) {
```

Comment:
That would certainly be a valuable addition and consistent with other tests like `testEmissionWavelengths`. The only reason for not adding it at this point is the size of the corresponding configuration PR fixing the mismatching configured values.

Can we handle this via a follow-up PR once this addition is included and first review the number of changes to do across the configured filesets?